FDK/src/02.Input/CInputGamepad.cs

```diff
@@ -132,6 +142,12 @@ private void Joystick_ButtonUp(IGamepad joystick, Button button) {
 				ButtonStates[(int)button.Name].Item1 = false;
 			}
 		}
+
+		private void Gamepad_ThumbstickMoved(IGamepad joystick, Thumbstick thumbstick) {
+		}
+
+		private void Gamepad_TriggerMoved(IGamepad joystick, Trigger trigger) {
+		}

```

Comment:
Currently dead code, should also follow `PascalCase` e.g. `GamepadThumbstickMoved`.